### PR TITLE
bt-60: add  loader

### LINKED ISSUE #60: Create Loader component
The loader should be a fullscreen container with a spinner in the center. The container should be white and the spinner is Primary color

### PR DESCRIPTION
**Pull request by mskmee**
image

### CODE REVIEW

File: mobile/src/bundles/common/components/loader/styles.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    wrapper: {
+        width: '100%',
+        height: '100%',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+});
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/loader/styles.ts]: Whole style object can be replaced with global styles

File: mobile/src/bundles/common/components/loader/styles.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    wrapper: {
+        width: '100%',
+        height: '100%',
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/loader/styles.ts]:
```suggestion
        flex: 1
```

File: mobile/src/bundles/common/components/loader/styles.ts (hunk 1 above)
Comment (mskmee) [reply mobile/src/bundles/common/components/loader/styles.ts]: fix

File: mobile/src/bundles/common/components/loader/loader.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import { Color } from '~/bundles/common/enums/enums';
+
+import { styles } from './styles';
+
+type LoaderProperties = {
+    size: 'small' | 'large';
+};
+const Loader: React.FC<LoaderProperties> = ({ size }) => {
+    return (
+        <View style={styles.wrapper}>
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/loader/loader.tsx]:
```suggestion
        <View style={[globalStyles.flex1, gobalStyles.justify..., ...]}>
```

File: mobile/src/bundles/common/components/loader/styles.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+const styles = StyleSheet.create({
+    wrapper: {
+        flex: 1,
+        justifyContent: AlignmentValue.CENTER,
+        alignItems: AlignmentValue.CENTER,
+    },
+});
+
+export { styles };
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/loader/styles.ts]:
```suggestion

```

File: mobile/src/bundles/common/components/components.tsx [hunk 5]
```diff
@@ -1,4 +1,4 @@
 export { Button } from './button/button';
 export { Input } from './input/input';
 export { Link } from './link/link';
-export { Text, View } from 'react-native';
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/components.tsx]: re-export loader here also

File: mobile/src/bundles/common/components/loader/loader.tsx (hunk 3 above)
Comment (mskmee) [reply mobile/src/bundles/common/components/loader/loader.tsx]: fix

File: mobile/src/bundles/common/components/loader/styles.ts (hunk 4 above)
Comment (mskmee) [reply mobile/src/bundles/common/components/loader/styles.ts]: fix

File: mobile/src/bundles/common/components/components.tsx (hunk 5 above)
Comment (mskmee) [reply mobile/src/bundles/common/components/components.tsx]: fix

**Review by katerynakostikova — APPROVED** [review]

File: .gitignore
Comment (nikita-remeslov) [inline .gitignore:1]: EOL

**Review by nikita-remeslov — APPROVED** [review]

File: .gitignore
Comment (nikita-remeslov) [reply .gitignore:1]: @mskmee waiting for fix

**Pull request merged**